webroot/results/admin/fix_results.php

```diff
@@ -330,6 +331,16 @@ function somethingChanged()
     };
     $('#check-results').attr("href", "check_results.php?" + $.param(params));
   }
+  if(!lastCompetitionId) {
+    $('#check-rounds').hide();
```

Comment:
You can try something like `$('#check-rounds').parent().hide()` so that the whole `li` will be hide. This is the same for the below `show()` method.